.github/workflows/deploy-changed-cf.yaml

```diff
@@ -305,6 +305,8 @@ jobs:
                 CREATE ROLE dev_role WITH LOGIN;
                 GRANT USAGE ON SCHEMA public TO api_role;
                 GRANT USAGE, SELECT ON ALL SEQUENCES IN SCHEMA public to api_role;
+                GRANT USAGE ON SCHEMA aws_lambda TO api_role;
+                GRANT EXECUTE ON ALL FUNCTIONS IN SCHEMA aws_lambda TO api_role;
```

Comment:
This would be re run correctly on the existing `-default ` clusters?

Reply:
Also, by the way, do you think this is all we will need in the case we want to add more lambdas in the future that perform similar cleanup for other s3 buckets?

Reply:
These permissions are for the `api_role` to trigger lambda. The lambda is the one that accesses S3. So permissions to access the additional buckets should be defined in the lambda permissions, not here.

Reply:
Also this other part? 
> This would be re run correctly on the existing -default clusters?

Just merging this will be enough?

Reply:
Might need to manually patch. I don't yet know what happens to RDS if we try to create a new user while there already is existing user.